module-bsp/board/rt1051/puretx/hal/battery_charger/CurrentMeasurementScope.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "CurrentMeasurementScope.hpp"
+
+extern "C"
+{
+#include "FreeRTOS.h"
+}
+#include <timers.h>
+#include <bsp/battery_charger/battery_charger.hpp>
+#include <ticks.hpp>
+
+namespace hal::battery::CurrentMeasurementScope
+{
+    namespace
+    {
+        constexpr auto samplingTime_MS = 100;
```

Comment:
consider using `ms` suffix and `std::duration`'s count

Reply:
done with chrono